specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json

```diff
@@ -792,6 +798,30 @@
         }
       }
     },
+    "ExtractiveSummarizationTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
```

Comment:
consider adding `loggingOptOut` parameter similar to other tasks.

Reply:
Thanks for catching this.  I actually just opened a new PR for this because it seems that my master branch in my fork somehow got polluted and additional files keep getting added that I didn't intend to add. 

I've resolved this in the new PR [here](https://github.com/Azure/azure-rest-api-specs/pull/15159).